### ISSUE TITLE
Implement creation of media tags in contentMetadata

### ISSUE THREAD

**Issue by peetucket** [sul-dlss/media#26]
This will likely get implemented in the robot framework.  We need to create media specific metadata into contentMetadata necessary for embed player to work correctly.

**Comment by LynnMcRae** [comment]
These are <audioData> and <videoData> tags in contentMetadata, parallel to <imageData>.  See https://consul.stanford.edu/display/chimera/DOR+content+types%2C+resource+types+and+interpretive+metadata

**Comment by hannahfrost** [comment]
So duration won't show in the player until the duration data is in contentMetadata, and we don't plan to tackle that as part of this first media work cycle.

**Comment by LynnMcRae** [comment]
this sprint vs this work cycle?  Sorry, I still get confused !

I think dimensions for video is a requirement, so duration for that would/could go in at the same time. Audio is perhaps a separate effort.

**Comment by hannahfrost** [comment]
sorry, it should have said "work cycle" (now corrected above).

Dimensions for video is a requirement before going to production? What happens if dimensions data is not there? Is there a ticket and/or specs for the required data for video? Do we need to remediate the existing accessioned video content so that it works with the new media features before going to production? (This is related to https://github.com/sul-dlss/media/issues/107)

**Comment by LynnMcRae** [comment]
Yes, #107 is where I got my assumptions. I don't know the implications of lack of dimensions (default to 400px tall)?  I also believed the remediation would be something Ben and I could work on if the details are in the technical metadata, but no analysis yet.

**Comment by andrewjbtw** [comment]
I think this can be considered complete since media streaming has been in production for years and contentMetadata was changed to structural metadata years ago.
